### ISSUE TITLE
Sheet: npe when putting null value

### ISSUE THREAD

**Issue by eduarddrenth** [primefaces-extensions/primefaces-extensions#886]
### Describe the bug

PR #852 introduced an NPE on `Sheet.setLocalValue(Sheet.java:294)`, ConcurrentHashMap does not allow null for value:
```
    final V putVal(K key, V value, boolean onlyIfAbsent) {
        if (key == null || value == null) throw new NullPointerException();
```

### Expected behavior

Trying to put a null value into localValues should not result in an npe. Instead:

1. nothing should be put, or
2. when the key is present the value should be removed

Before PR #852 null values were allowed in HasMap, so the second option seems the better one.

### PrimeFaces Extensions version

11.0.5-SNAPSHOT

### JSF implementation

Mojarra

### JSF version

2.3

### Browser(s)

all

**Comment by melloware** [comment]
I assigned it to you thanks for reporting.  I agree if NULL the value should be removed.

**Comment by melloware** [comment]
I have another fix to make tomorrow then I will release 11.0.6

**Comment by eduarddrenth** [comment]
thnx, I'll wait for it

Op di 19 jul. 2022 23:50 schreef Melloware ***@***.***>:
